common/script/index.coffee

```diff
@@ -755,7 +755,7 @@ api.wrap = (user, main=true) ->
         if user.balance < 1
           return cb? {code:401,message: i18n.t('notEnoughGems', req.language)}
         else
-          user.balance--
+          user.balance -= 1
```

Comment:
Oh, in this case, it's just to keep readability consistent with the other decrements, e.g. https://github.com/HabitRPG/habitrpg/pull/4915/files#diff-9212b31a96588a139c40288e0155f9caR786

In general though, I like to avoid `++` / `--` operators because their behavior isn't always immediately intuitive as you're reading through code (especially in forced-return-value languages like CoffeeScript). As a trivial example, `var i = 2; console.log(i++);` will output 2, and set i to 3 (in contrast with `var i = 2; console.log(++i);`

For folks coming from C, the difference is obvious, but some modern languages don't allow that operator (Ruby being a notable example - especially as it was a big influence on CoffeeScript), so this can be confusing to folks.

*(for more rambling on `++`, see this snippet from [Douglas Crockford's talk](https://youtu.be/47Ceot8yqeI?t=1h9m16s))*

Anyway, in this case, I think it makes more sense to use `-=`, because we're using it in the `releasePets` and `releaseMounts` methods (because we need to decrement by a different value). If it's a point of contention though, I can certainly remove it :)